spec/lib/open_food_network/enterprise_injection_data_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'open_food_network/enterprise_injection_data'
+
+RSpec.describe OpenFoodNetwork::EnterpriseInjectionData do
+  let(:enterprise1) { create :distributor_enterprise, with_payment_and_shipping: true }
+  let(:enterprise2) { create :distributor_enterprise, with_payment_and_shipping: true }
+  let(:enterprise3) { create :distributor_enterprise, with_payment_and_shipping: true }
+  let(:enterprise4) { create :distributor_enterprise, with_payment_and_shipping: true }
+
+  before do
+    [enterprise1, enterprise2, enterprise3].each do |ent|
+      create :open_order_cycle, distributors: [ent]
+    end
+  end
+
+  let!(:closed_oc) { create :closed_order_cycle, coordinator: enterprise4 }
+
+  context "when scoped to specific enterprises" do
+    let(:subject) {
+      described_class.new([enterprise1.id, enterprise2.id])
+    }
```

Comment:
FYI you can use the `subject` method instead :
```suggestion
    subject {
      described_class.new([enterprise1.id, enterprise2.id])
    }
```